packages/bits/src/lib/textbox/textbox-number/textbox-number.component.ts

```diff
@@ -262,7 +262,17 @@ export class TextboxNumberComponent implements ControlValueAccessor, NuiFormFiel
     }
 
     private isMetaKey(event: KeyboardEvent): boolean {
-        return event.ctrlKey || event.metaKey || event.code === "Backspace" || event.code === "Delete" || event.code === "Tab";
+        return (
+            event.ctrlKey ||
+            event.metaKey ||
+            event.code === "Backspace" ||
+            event.code === "Delete" ||
+            event.code === "Tab" ||
+            event.code === "ArrowRight" ||
+            event.code === "ArrowLeft" ||
+            event.code === "ArrowUp" ||
+            event.code === "ArrowDown"
```

Comment:
I think it would be good to use this enum here:

https://github.com/solarwinds/nova/blob/9513fdeeff76041946aafb9c7fcb4ebe502d3602/packages/bits/src/constants/keycode.constants.ts#L24